htdocs/js/firefoxDates.js

```diff
@@ -0,0 +1,37 @@
+$(document).ready(function() {
+  // If <input type="date" /> is not supported (i.e. Firefox), load
```

Comment:
"firefoxDates" is a bad name for this file. It should just be called something like "polyfills" since 1) it applies to anything that doesn't support a date input type and 2) that way we can extend it for any other polyfills that we need to add in the future..